src/main/java/com/LP2/EventScheduler/controller/EventController.java

```diff
@@ -55,6 +56,14 @@ public ResponseEntity<MessageResponse> joinTheEvent(
     ) {
         return new ResponseEntity<>(this.eventService.joinEvent(eventId, joinData, user), HttpStatus.CREATED);
     }
+    @DeleteMapping(path = "/{eventId}/delete")
+    public ResponseEntity<MessageResponse> removeEvent(
+            @PathVariable("eventId") UUID eventId,
+            @RequestAttribute("user") User user,
+            @Valid @RequestBody DeleteEventDTO deleteData
```

Comment:
No tiene sentido que pongas un DTO, ya que el id del evento ya lo obtienes por `@PathVariable("eventId") UUID eventId`

```suggestion
            @PathVariable("eventId") UUID eventId,
            @RequestAttribute("user") User user
```